alerting/src/main/java/org/hawkular/metrics/alerting/InsertedDataSubscriber.java

```diff
@@ -56,15 +55,20 @@
 public class InsertedDataSubscriber {
     private static final Logger log = Logger.getLogger(InsertedDataSubscriber.class);
 
+    static final String PREFIX_AVAILABILITY = "hm_a_";
```

Comment:
I'm not sure where this is best documented.  I'll add something here just to explain why the prefixes exist and why we have a convention.  But it may need to be in user docs somewhere in metrics.